src/main/kotlin/ExerciseNr23.kt

```diff
@@ -0,0 +1,45 @@
+/*Given a positive integer n implement a function that returns a list representing the generatePyramid shape with n levels.
+The function should console log a generatePyramid shape with N levels using the # character.
+Make sure the generatePyramid has spaces on both the left and right-hand sides.
+Examples:
+pyramid(1)
+    '#'
+
+pyramid(2)
+    ' # '
+    '###'
+
+pyramid(3)
+    '  #  '
+    ' ### '
+    '#####'
+*/
+
+
+//fun main(args: Array<String>) {
+//    println(pyramid(4))
+//}
+
+fun pyramid(rows: Int) {
```

Comment:
for pyramid(5) it prints me 
                   #
               ###
           #####
       #######
  #########
      kotlin.Unit